.github/workflows/on-merged-pr.yml

```diff
@@ -12,4 +12,5 @@ jobs:
     steps:
       - name: 'Trigger obscuro-test local testnet tun'
         run: |
-          curl -XPOST -H "Authorization: Bearer ${{ secrets.GH_TOKEN }}" -H "Accept:application/vnd.github" -H "Content-Type:application/json" https://api.github.com/repos/obscuronet/obscuro-test/actions/workflows/local_tests.yml/dispatches --data '{"pr_number": "${{ github.event.number }}" }'
+          curl -XPOST -H "Authorization: Bearer ${{ secrets.GH_TOKEN }}" -H "Accept:application/vnd.github" -H "Content-Type:application/json" https://api.github.com/repos/obscuronet/obscuro-test/dispatches --data '{ "event_type": "merged_pull_request", "client_payload": { "number": ${{ github.event.number }} } }'
```

Comment:
Tripple checking

From `.../obscuro-test/actions/workflows/local_tests.yml/dispatches` to `/obscuro-test/dispatches` 

That's correct yeah ?
I thought it would need the workflow path 🤔

Reply:
https://stackoverflow.com/questions/68147899/whats-is-the-difference-between-repository-dispatch-and-workflow-dispatch-in-git

Reply:
Yes this is now using a repository dispatch, not a workflow dispatch - there are differences between them like the ability to include the event type, and a custom payload field.

Reply:
Ohh, nice feature, wasn't aware of it. 👌